hs_file_types/models/base.py

```diff
@@ -886,6 +921,11 @@ def get_aggregation_class_name(self):
         """Return the class name of the logical type (aggregation type)"""
         return self.__class__.__name__
 
+    @property
+    def is_fileset(self):
+        """Return True if this aggregation is a fileset aggregation, otherwise False"""
+        return self.get_aggregation_class_name() == 'FileSetLogicalFile'
```

Comment:
maybe compare the class instead of a string representation of the class name?

Reply:
To compare it with class I have to import the class within this function. I would prefer not do an import here.